spec/helper/versioning_helper_spec.rb

```diff
@@ -309,6 +315,8 @@
       )
       expect(changelog).to eq("### New Features\n" \
                               "* added a log when `autoSyncPurchases` is disabled (#1749) via aboedo (@aboedo)\n" \
+                              "### Paywalls\n" \
```

Comment:
Should we name this `RevenueCatUI` instead?

Reply:
Hmm I would match it with the name of the library, so there is less confusion. So I would say yes?